node/_tools/test/parallel/test-fs-read.js

```diff
@@ -84,22 +84,22 @@ assert.throws(
   }
 );
 
-// TODO(wafuwafu13): Enable this
+// don't have to pass because throwing ERR_INVALID_ARG_TYPE.
```

Comment:
I'm not sure what this comment means. Are you saying we should never uncomment this code? If that's the case we'd never be able to automatically update this test in the future.

Reply:
I think this test is not ideal and [created PR](https://github.com/nodejs/node/pull/44769), but fixed so that this test passes for compatibility. https://github.com/denoland/deno_std/pull/2692/commits/1e4c61d6214088c08e428315532008467f446e35